test/Microsoft.ML.TorchSharp.Tests/Microsoft.ML.TorchSharp.Tests.csproj

```diff
@@ -3,6 +3,9 @@
     <AssemblyName>Microsoft.ML.TorchSharp.Tests</AssemblyName>
     <StrongNameKeyId>Test</StrongNameKeyId>
 
+    <!-- Remove once we have resolved the TorchSharp issue. -->
+    <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
```

Comment:
>ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch [](http://example.com/codeflow?start=5&length=54)

What this exactly does?

Reply:
It's required to avoid `MSB3270`, but I thought this wasn't needed on the 8.0 SDK - I thought that warning was completely removed.

Reply:
Nope. Its still required unless I am doing something wrong. Throws an error without that.